create-layla-coin/run.py

```diff
@@ -0,0 +1,46 @@
+# LaylaCoin Tutorial
+# Single Script Executable
+
+# Imports
```

Comment:
Since you are adding these at the appropriate steps below and the flow would be for a user to uncomment each step as they go along, then I would remove them. Right now they aren't really serving any purpose.

Reply:
Got it - thanks! I removed the comments from the functional syntax.